src/features/Tiles/MovieTiles/index.js

```diff
@@ -20,16 +20,17 @@ import {
 } from "./styled";
 import { Rate } from "../../Rate";
 import { toMovie } from "../../../core/App/routes";
-import {  selectMoviesData, selectMoviesSearchQuery } from "../../PopularMovies/moviesSlice";
+import { selectMoviesData, selectMoviesSearchQuery } from "../../PopularMovies/moviesSlice";
 import { ShowMoreButton } from "../ShowMoreButton";
 import { StyledSection } from "../../../common/MovieAndPersonSection";
 
 const MovieTiles = ({ title }) => {
     const moviesData = useSelector(selectMoviesData);
     const searchQuery = useSelector(selectMoviesSearchQuery);
+    const configuration = useSelector(selectConfiguration);
 
-    const imageURL = "http://image.tmdb.org/t/p/";
-    const size = "w342";
+    const imageURL = configuration.images.base_url;
+    const size = configuration.images.poster_sizes[3];
```

Comment:
I find it rathe negative. The App its loading the data from the array index - if there would changed data inside -  it would work anyway, but results can be different.

Reply:
Yes, you are right. It would still work even when the data is changed but for example here you need at least 300w picture and you have no garancy that in configuration.images.poster_sizes[3] you will get big enough picture in the future. It's not such a big deal but I think there is still a risk we can end up with too small (or too big) picture with your solution.